src/v2/auth.rs

```diff
@@ -319,12 +321,29 @@ mod tests {
         let service = "sat-r220-02.lab.eng.rdu2.redhat.com";
         let scope = "repository:registry:pull,push";
 
-        let header_value = HeaderValue::from_str(&format!(
-            r#"Bearer realm="{}",service="{}",scope="{}""#,
-            realm, service, scope
-        )).expect("this statically known header value only contains ASCII chars so it is correct header value");
-
-        let content = WwwAuthenticateHeaderContent::from_www_authentication_header(header_value)?;
+        for header_value in [
+            HeaderValue::from_str(&format!(
+                r#"Bearer realm="{}",service="{}",scope="{}""#,
+                realm, service, scope
+            )).unwrap(),
+            HeaderValue::from_str(&format!(
+                r#"bearer realm="{}",service="{}",scope="{}""#,
+                realm, service, scope
+            )).unwrap(),
+            HeaderValue::from_str(&format!(
+                r#"BEARER realm="{}",service="{}",scope="{}""#,
+                realm, service, scope
+            )).unwrap(),
+            HeaderValue::from_str(&format!(
+                r#"Bearer Realm="{}",Service="{}",Scope="{}""#,
+                realm, service, scope
+            )).unwrap(),
+            HeaderValue::from_str(&format!(
+                r#"Bearer REALM="{}",SERVICE="{}",SCOPE="{}""#,
+                realm, service, scope
+            )).unwrap()
+        ] {
```

Comment:
```suggestion
].iter() {
```
To support Rust 1.46